tools/README.md

```diff
@@ -6,6 +6,7 @@ should be considered experimental.
 
 * [`setup-unit`](#setup-unit)
 * [`unitc`](#unitc)
+* [`cli`](#cli)
```

Comment:
Shouldn't it be `tools/unitctl`?

`cli` is a bit generic and we have three of them...

Reply:
Hmm, I see there is `tools/cli/unitctl` and I guess `tools/unitctl/unitctl` _may_ seem a little weird.

I'd suggest switching `cli` and `unitctl` around, i.e

`tools/unitctl/cli`